File: PY/bill.py
#Bill

class Bill:
    
    def __init__(self, bill_id, title, descp) -> None:
        self.bill_id = bill_id
        self.title = title
        self.descp = descp
        self.loksabha_pass = False
        self.rajyasabha_pass = False
        
    def bill_loksabha(self, loksabha_pass):
        self.loksabha_pass = loksabha_pass
    
    def bill_rajyasabha(self, rajyasabha_pass):
        self.rajyasabha_pass = rajyasabha_pass
        
    def bill_can_be_passed(self):
        return self.loksabha_pass and self.rajyasabha_pass
    
    def get_bill_id(self):
        return self.bill_id
    
    def get_bill(self):
        return self.bill_id+"\n"+self.title.upper()+"\n"+self.descp

File: C#/Bill.cs

public class Bill{
    private string billId;
    private string title;
    private string description;

    private bool loksabhaPass;
    private bool rajyasabhaPass;

    public Bill(string billID, string title, string description){
        this.billId = billID;
        this.title = title;
        this.description = description;
    }

    public void billLokSabha(bool loksabhaPass){
        this.loksabhaPass = loksabhaPass;
    }

    public void billRajyaSabha(bool rajyasabhaPass){
        this.rajyasabhaPass = rajyasabhaPass;
    }

    public bool billCanBePassed(){
        return this.loksabhaPass && this.rajyasabhaPass;
    }

    public string getBillID(){
        return this.billId;
    }

    public string getBill(){
        return this.billId+"\n"+this.title + "\n" + this.description;
    }

}